### DISCUSSION TITLE
iOS Required Reason API / App Privacy Manifest declaration

### DISCUSSION THREAD

**Discussion by kgunderman** [invertase/react-native-firebase#7488] [General]
According to the apple docs: https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api

Starting in spring of 2024, libraries must declare the reasons they are using required reason API within the privacy manifest. As a library that uses the required reason API, are there plans to update the privacy manifest to include this reasoning?

**Comment by mikehardy** [comment]
This should be posed in firebase-ios-sdk - we just wrap the SDKs here and expose javascript bindings for the convenience of the react-native ecosystem, but do not have control over the underlying SDK

A quick search over there (here is one, https://github.com/firebase/firebase-ios-sdk/issues?q=is%3Aissue+NSPrivacyAccessedAPIType) did not turn up any currently open issues so it is likely a good idea for you to open one, and it would be great if you can link it back here

**Reply by kgunderman** [reply]
Thanks :) I've opened up an issue over there: https://github.com/firebase/firebase-ios-sdk/issues/12163

**Comment by kgunderman** [comment]
Looks like this is already being tracked and worked on: https://github.com/firebase/firebase-ios-sdk/issues/11490

**Reply by mikehardy** [reply]
yeah, didn't have quite the right search terms, but luckily firebase-ios-sdk is on top of their game :-). Subscribed to the existing issue now myself. Cheers
